### ISSUE TITLE
Violation warnings about non-passive event listeners

### ISSUE THREAD

**Issue by PropGit** [parallaxinc/solo#267]
While helping an instructor this morning who was having problems where the port list wouldn't populate (yet BP Launcher and Solo were reporting _connected_), I tested on my side and found that Solo is throwing multiple warnings, some of which are tagged as Violations.

![image](https://user-images.githubusercontent.com/1266318/73958307-950e1100-48bc-11ea-95eb-d79ceb3785f4.png)

NOTE: This may or may not be what was causing problems for the instructor; at the moment, he is back in operation again for an unknown reason.

**Comment by PropGit** [comment]
This was captured from Chrome v79.0.3945.130 on Win7.

**Comment by zfi** [comment]
The mouse and mouse-wheel event messages are part of the performance metrics package. They are recommendations to improve performance. It appears that these are coming from Blockly core.

The last two are a little more concerning. @PropGit, do you have a copy of the submitted image that does not truncate the line number in blockly.html:1???  ?

**Comment by PropGit** [comment]
That's actually not a truncation... it's reporting line 1.  Something revealing is that the reported source changes depending on where we're at in the app.

For example: when on the landing page, it reports this:
![image](https://user-images.githubusercontent.com/1266318/73971675-26d54880-48d4-11ea-97e3-2c76534b054c.png)

And after reaching the new project dialog (or editor page) it reports this:
![image](https://user-images.githubusercontent.com/1266318/73971716-3a80af00-48d4-11ea-85ed-eee493e6cd54.png)

**Comment by PropGit** [comment]
**IMPORTANT**

I just tried on Win10 with v80.0.3987.87 and do not see any of those yellow messages in the console.

**Comment by PropGit** [comment]
I'm seeing debug output on the Solo console that I did not see in my earlier tests. It appears it's logging the object for the compiler results.
![image](https://user-images.githubusercontent.com/1266318/73974054-89304800-48d8-11ea-9524-58059a7021aa.png)

**Comment by PropGit** [comment]
This may be a non-issue ([see post](https://github.com/parallaxinc/solo/issues/267#issuecomment-583083214)) but will leave open for a while until we determine for sure.

**Comment by PropGit** [comment]
**DETAIL UPDATE**

The "cross-site resource" warnings can be ignored... they are indeed caused by one or more extensions installed on my Chrome browser.  "Stencil" is an extension I've installed but haven't subscribed to, and the "getstencil.com" items are related to that extension trying to advertise to me.  I've achieved getting those messages again in Win 10 by signing into Chrome, then refreshing the browser.

The [Violation] messages may also be related to an extension on my side, but I haven't proven that.  I can not make them occur on Win 10 in my tests.
